src/html/utils.ts

```diff
@@ -11,6 +11,17 @@ export abstract class HtmlUtils {
                 return attribute.value;
             }
         }
+
+        return null;
+    }
+
+    public static getNormalizedAttributeValue(element: Element, attributeName: string, options: IContentOptions): string | null {
+        for (let attribute of element.attrs) {
```

Comment:
Instead of repeating the same for loop and if statement I suggest you call `getAttributeValue` instead and then normalize the returned value. Just make sure to handle `null`.